client/allocrunner/taskrunner/envoybootstrap_hook.go

```diff
@@ -148,3 +166,44 @@ func (h *envoyBootstrapHook) Prestart(ctx context.Context, req *interfaces.TaskP
 	resp.Done = true
 	return nil
 }
+
+type envoyBootstrapArgs struct {
+	sidecarFor     string
+	grpcAddr       string
+	consulHTTPAddr string
+	siToken        string
+}
+
+func (e envoyBootstrapArgs) args() []string {
```

Comment:
I don't think we've done this pattern before so make sure to comment it and/or the struct.